packages/loader/src/gltf/parser/GLTFSceneParser.ts

```
@@ -45,24 +46,24 @@
       glTFResource._defaultSceneRoot = sceneRoot;
     }
 
-    const promises = new Array<Promise<void[]>>();
+    const promises = new Array<AssetPromise<void[]>>();
```

Biome (1.9.4)

[error] void is confusing outside a return type or a type parameter.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)